repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py

```diff
@@ -186,6 +187,8 @@ def version2_check(info):
             continue
         if version.matches_target_version('> 8.6', '< 9.0') or version.matches_target_version('> 9.0'):
             # if a target release is >=8.8 or >=9.2, the SAP HANA and RHEL versions compatibility is not checked
+            # TODO(mmatuska): We don't know whether the check will be skipped on RHEL 10 (9to10) yet,
+            # update the method accoridingly then
```

Comment:
I will create a ticket to update checksaphana actor for RHEL 10 as we need to modify the actor even if the check is skipped because there are versions mentioned in the report.